elyra/pipeline/handlers.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright 2018-2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+
+from notebook.base.handlers import APIHandler
+from .parser import PipelineParser
+from .processor import PipelineProcessorManager
+from ..util.http import HttpErrorMixin
+
+
+class PipelineExportHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose REST API to export pipelines"""
+
+    def get(self):
```

Comment:
Should this exist at all?  I think a 404 or however a bogus request is handled is sufficient. I think the same can be said about the submission endpoint as well.

Reply:
With the get method you get the following when accessing the API directly:

```
{

    "title": "Operation not supported."

}
```

Without it:
```
{

    "reason": "Method Not Allowed",
    "message": "",
    "timestamp": "2020-04-19 12:33:59"

}
```

Any preference?

Reply:
Understood.  I don't know which is more correct.  To me, it seems like implementing GET when there are no resources to retrieve feels misleading and not implementing the method at all feels more correct.  But I just haven't been around REST enough to know what's considered the best practice.